packages/ripple-binary-codec/src/enums/constants.ts

```diff
@@ -2,3 +2,4 @@ export const TYPE_WIDTH = 2
 export const LEDGER_ENTRY_WIDTH = 2
 export const TRANSACTION_TYPE_WIDTH = 2
 export const TRANSACTION_RESULT_WIDTH = 1
+export const DELEGATABLE_PERMISSIONS_WIDTH = 4
```

Comment:
```suggestion
export const LEDGER_ENTRY_WIDTH = 2 // UInt16
export const TRANSACTION_TYPE_WIDTH = 2 // UInt16
export const TRANSACTION_RESULT_WIDTH = 1 // UInt8
export const DELEGATABLE_PERMISSIONS_WIDTH = 4 // UInt32
```
To hopefully avoid the issue you had in the future